src/swsssdk/configdb.py

```diff
@@ -101,16 +102,28 @@ def __raw_to_typed(self, raw_data):
         if raw_data == None:
             return None
         typed_data = {}
-        for key in raw_data:
+        for raw_key in raw_data:
+            key = raw_key
+            if PY3K:
```

Comment:
>PY3K [](start = 15, length = 4)

Why we need to check python3? Is it possible not to?

Reply:
Library supposes to work with Python 2 and 3. Current implementation doesn't handle encoding correctly in Python 3. Python 2 has different API to works with encoding than Python 3. So it is not possible to not check.